jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/editor/CodeWhispererEditorListener.kt

```diff
@@ -18,6 +21,18 @@ class CodeWhispererEditorListener : EditorFactoryListener {
             object : DocumentListener {
                 override fun documentChanged(event: DocumentEvent) {
                     CodeWhispererInvocationStatus.getInstance().documentChanged()
+                    // When open a file for the first time, IDE will also emit DocumentEvent for loading with `isWholeTextReplaced = true`
+                    // Added this condition to filter out those events
```

Comment:
I tried debugging but this wasn't the case for me. Can you make sure if you are filtering this correctly?

Reply:
![image (1)](https://user-images.githubusercontent.com/96078566/180092100-d77d5d09-647b-4842-9dc8-5a61cd49b674.png)

Reply:
^^ is what I get when I just open the IDE sandbox on loading the codewhisperer welcome.md file.